examples/solids/qfunctions/FSCurrent-NH2.h

```diff
@@ -65,7 +66,8 @@ static inline CeedScalar log1p_series_shifted(CeedScalar x) {
   sum += y / 5;
   y *= y2;
   sum += y / 7;
-  return 2 * sum;
+  *log1p_shifted = 2 * sum;
```

Comment:
That's the issue here, but I don't think there's a reason the macro needs to capture the name. We needed it for `CEED_QFUNCTION` because we had to create the location string in addition to the function.

Reply:
Good point with the name.